src/VDPDataStore.cpp

```diff
@@ -53,12 +53,41 @@ class VDPDataStoreVRAMSizeCheck : public SimpleCommand
 	VDPDataStore& dataStore;
 };
 
+class VDPDataStoreDebuggableChecks : public SimpleCommand
+{
+public:
+	VDPDataStoreDebuggableChecks(VDPDataStore& dataStore_)
+		: SimpleCommand("debug_check_debuggables {{VDP register latch status} {VDP palette latch status} {VDP data latch value}}")
+		, dataStore(dataStore_)
+	{
+	}
+
+	void replyOk(const QString& message) override
+	{
+		dataStore.registerLatchAvailable = message.contains("{1}");
+		dataStore.paletteLatchAvailable = message.contains("{2}");
+		dataStore.dataLatchAvailable = message.contains("{3}");
```

Comment:
If you would switch to the alternative `debug_check_debuggables` proc (see above). Then this implementation needs to change.
Ideally we would have a proper Tcl list parser (*1). But in this case the resulting list will have a very simple structures: `0` and `1` characters separated by spaces. So you could do something like:
```
assert(message.size() == 5);
dataStore.registerLatchAvailable = message[0] == '1';
```
(I'm assuming `QString` allows the same kind of operations as `std::string`).

--

(*1)  Just a remark on the side. No need to do anything like this in this commit.

With a proper Tcl list parser I mean a function like:
```std::vector<std::string> parseTclList(const std::string& list);```
(or with `QString` and/or `QList`, `QStringList`).

This utility would take care of splitting the input string on spaces, taking all Tcl escaping rules (braces, backspace, quotes) into account.

Such a utility could also be used e.g. in `DebuggerForm::setDebuggables()` to make the current ad-hoc parsing simpler and more robust.